_topic_maps/_topic_map_ms.yml

```diff
@@ -58,6 +58,15 @@ Topics:
 - Name: Greenboot health check
   File: microshift-greenboot
 ---
+Name: Updating clusters
```

Comment:
I'm not sure whether it's relevant to point this out, but it looks like there is some documentation for "Checking your Microshift version" included in the "Troubleshooting" docs: 

Name: Troubleshooting
Dir: microshift_troubleshooting
Distros: microshift
Topics:
- Name: Checking your version
  File: microshift-version
- Name: Additional information
  File: microshift-things-to-know

Would some things be shared? Would that content be updated/removed? 

Not sure. Just pointing it out :D

Reply:
@stevsmit It is the same assembly, yes. I didn't think I could reuse an assembly like a module because it would have the same ID and context? If there's an easier way, I am all for it.